README.md

```diff
@@ -1,69 +1,89 @@
 
-Command Parameter Structure: usb!command!command parameter (optional)
+Command Parameter Structure
+```
+usb!!<command>!!<command parameter (optional)>
+```
 
-2.3. Managing Filesystem
+### Managing Filesystem
 
 There are a couple commands which should be sent in order to allow for the app to understand how it should create the filesystem and the contents that are copied. 
+___
 
-Command: usb!startfilesystem
+_Command_: 
+```
+usb!!startfilesystem
+```
 
-Description: Sends a command to notify the app to acknowledge when to start listening to the addasset UDP commands below. 
+_Description_: Sends a command to notify the app to acknowledge when to start listening to the addasset UDP commands below. 
+___
 
-Command: usb!addasset!<filename or directory>
-  
-Description: Sends a command to notify the app what is the name of a file or directory that should be copied to the usb drive. This command can be sent multiple times.
+_Command_: 
+```
+usb!!addasset!!<filename or directory>
+```
 
-Note: The current expectation of this example is that the source dependencies live on the root of the SD card.
+_Description_: Sends a command to notify the app what is the name of a file or directory that should be copied to the usb drive. This command can be sent multiple times.
 
-Command: usb!closefilesystem
+__Note__: The current expectation of this example is that the source dependencies live on the root of the SD card.
+___
 
-Description: Sends a command to notify the app to close and build the filesystem. The size for this example is 1GB, although this can be updated in the Javascript or additional support can be added to manage this at a more granular level using either “User Variables” or additional UDP commands.
+_Command_: 
+```
+usb!!closefilesystem
+```
 
-2.4. Mount/Unmount filesystem
+_Description_: Sends a command to notify the app to close and build the filesystem. The size for this example is 1GB, although this can be updated in the Javascript or additional support can be added to manage this at a more granular level using either “User Variables” or additional UDP commands.
+___
 
-2.4.1. Mount
+### Mount filesystem
 
-Command: usb!mount
+_Command_: 
+```
+usb!!mount
+```
 
-Description: Sends the command to mount the USB filesystem to the external client via the USB-C-to-USB-A cable.
+_Description_: Sends the command to mount the USB filesystem to the external client via the USB-C-to-USB-A cable.
+___
 
-2.4.2. Unmount
+### Unmount filesystem
 
-Command: usb!unmount
+_Command_: 
+```
+usb!!unmount
+```
 
-Description: Sends the command to unmount the USB filesystem to the external client via the USB-C-to-USB-A cable.
+_Description_: Sends the command to unmount the USB filesystem to the external client via the USB-C-to-USB-A cable.
```

Comment:
Any commands that can check or delete assets?

Reply:
Neither a check asset or delete asset were created. These could be added if there is use for them.